src/views/Deposit/PrizePoolNetworkCarousel/UpcomingPrize.tsx

```diff
@@ -61,10 +62,26 @@ const AmountOfPrizes = (props) => {
     }
   }, [queryResults])
 
+  const weeklyAmountOfPrizes = amountOfPrizes * 7
```

Comment:
Looks like on load this will flicker from "Lots of prizes" to the correct amount. We should add an empty loading state with the same height as the text so the page content doesn't bounce around and it renders the correct string the first time

Reply:
Hid the text while loading through c882de6!